### DISCUSSION TITLE
Flutter IOS build timeout

### DISCUSSION THREAD

**Discussion by vadimbarda** [codemagic-ci-cd/codemagic-docs#966] [Q&A] [answered]
Hi, team. I'm developer and I have got Flutter application. I worm on PC, but also have a Virtual Machine with MacOS Monterey to build this application on my real IPad. Some times ago I decided to use Codemagic to build my application and publish for testing. I don't have issues with Android. But I can't build for IOS. There is no issues on build. It is just build during 2 hrs and failed by timeout. It is not a huge project. Android build is about 5 minutes. IOS release build on my local Mac Virtual Machine is about 2 minutes. Any suggestions? It is one of my timeouted build example ID 63288cc85ef2b12f5a3d824e. Thank you in advance

**Comment by himesh-cm** [comment, marked as answer]
Hi @vadimbarda 
Thanks for sharing the info, i will check and get back to you.

**Reply by himesh-cm** [reply]
Hi @vadimbarda 
Please delete your previous cache as well.

Please add --verbose to your flutter iOS build command and share the build id.
Currently, the info is limited to finding out the root cause of the issue.

**Reply by vadimbarda** [reply]
Hi @himesh-cm. Thanks for your suggestion to add --verbose to build command. Now I see issue in build. My issue was in incorrect adding GoogleService-Info.plist to the project. It has reference to my local Download folder. Now it works fine.
